src/rgw/rgw_http_errors.h

```diff
@@ -10,69 +10,69 @@ struct rgw_http_errors {
   int err_no;
   int http_ret;
   const char *s3_code;
+  const char *s3_message; // error message in DeleteMultiObjects.
```

Comment:
@Liuchang0812 (NIT) Please consider to keep PR title < 50 characters for future PRs.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits